components/activity_log_modal/activity_log_modal.tsx

```diff
@@ -2,64 +2,71 @@
 // See LICENSE.txt for license information.
 
 import $ from 'jquery';
-import PropTypes from 'prop-types';
 import React from 'react';
 import {Modal} from 'react-bootstrap';
 import {FormattedMessage} from 'react-intl';
 
+import {Session} from 'mattermost-redux/types/sessions';
+import {ActionFunc} from 'mattermost-redux/types/actions';
+
 import ActivityLog from 'components/activity_log_modal/components/activity_log';
-import LoadingScreen from 'components/loading_screen';
 
-export default class ActivityLogModal extends React.PureComponent {
-    static propTypes = {
+export type Props = {
 
-        /**
-         * The current user id
-         */
-        currentUserId: PropTypes.string.isRequired,
+    /**
+ * The current user id
+ */
```

Comment:
Can we fix this indent here? I don't know why this is not caught by the linter :thinking: 
```suggestion
    /**
     * The current user id
     */
```